adopt-a-dog/blog.html

```diff
@@ -0,0 +1,61 @@
+      <link href="https://fonts.googleapis.com/css?family=Indie+Flower" rel="stylesheet">
+      <link href="https://fonts.googleapis.com/css?family=PT+Sans" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+    <img class="logo" src="adoptadog-logo.svg" alt="Adopt a dog logo">
+    <img id="cart-icon" src="cart-icon.svg" alt="cart-icon">
+  </header>
+    <nav>
+        <a href="index.html">Home </a>  |
+        <a href="dogs.html">Dogs</a>   |
+        <a href="blog.html">Blog</a>
+    </nav>
+    <h1>Adopt A Dog Blog</h1>
+    <div class="blog-card">
+      <img class="Eric" src="images/patrick-hendry-221863-unsplash.JPG" alt="A woman and a Dog">
+      <h2>Traveling with your Dog</h2>
+      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt
+        ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+        laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+        voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+        non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+    </div>
+    <div class="blog-card">
+      <img class="Eric" src="images/matt-nelson-259365-unsplash.JPG" alt="Walking Dogs">
+      <h2>How to Walk multiple Dogs</h2>
+      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt
+        ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+        laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+        voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+        non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+    </div>
+    <div class="blog-card">
+      <img class="Eric" src="images/wyatt-ryan-367017-unsplash.JPG" alt="A girl and a Dog">
+      <h2>Teaching Your Dog to Fetch</h2>
+      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt
+        ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+        laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+        voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+        non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+    </div>
+
+  <footer>
+    <img class="logo" src="adoptadog-logo.svg" alt="Adopt a dog logo">
+    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+    <nav>
+      <a href="index.html">Home </a>  |
+      <a href="dogs.html">Dogs</a>   |
+      <a href="blog.html">Blog</a>
+    </nav>
+    <small>&copy; Copyright 2018, Adopt A Dog, LLC</small>
```

Comment:
I'd steer you away from using the small tag, but instead using a normal p or header tag, then adjusting the font size using CSS. It's technically correct, but we like to avoid <b>, <i>, and <small> tags.